### PR TITLE
Calendar and Product Description

### PR DESCRIPTION
**Pull request by jimena-garcia**
there are descriptions about the product and their specification and comparation with others apps

### CODE REVIEW

File: docs/Product Description/Similar Software.md
Comment (jeusmateo) [inline docs/Product Description/Similar Software.md:1]: Who is our project on the table?

File: docs/Product Description/Software Description.md
Comment (jeusmateo) [inline docs/Product Description/Software Description.md:1]: We must define the project name

**Review by jeusmateo — CHANGES_REQUESTED** [review]
Ok, but we must define the project name c:
Edit: we must change the project name, because a project is named similar

File: docs/Product Description/Similar Software.md
Comment (jimena-garcia) [reply docs/Product Description/Similar Software.md:1]: is the first column on the table

File: docs/Product Description/Software Description.md
Comment (jimena-garcia) [reply docs/Product Description/Software Description.md:1]: it's ready, line 3

**Review by jeusmateo — APPROVED** [review]
Ok

**Pull request merged**